config/webpack/loaders/devSiteCssFinder.js

```diff
@@ -0,0 +1,20 @@
+const findCss = (x) => {
```

Comment:
nit: can we name `devSiteCssFinder` to `devSiteDetermineCssFileName` or something to indicate it's returning a file name?

Also, should this file live in a utils since it's not a dev site loader?

Reply:
Updated file name and moved it to loaderUtils, I also updated the jest test names and location: [d830ef0](https://github.com/cerner/terra-dev-site/pull/281/commits/d830ef07a820fb004bc59f65cde8f20ee7052353#).